src/Aspire.Dashboard/Components/Resize/BrowserDimensionWatcher.razor.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace Aspire.Dashboard.Components.Resize;
+
+public class BrowserDimensionWatcher : ComponentBase
+{
+    [Parameter] public ViewportInformation? ViewportInformation { get; set; }
```

Comment:
nit: Repo convention is attribute on the line above the method/property.

Reply:
changed